lib/evmone/instructions.hpp

```diff
@@ -820,6 +820,40 @@ inline void swap(StackTop stack) noexcept
     a[3] = t3;
 }
 
+inline code_iterator dupn(StackTop stack, ExecutionState& state, code_iterator pos) noexcept
+{
+    const auto n = pos[1];
```

Comment:
Looks correct, but a bit confusing when you compare it to EIP spec.
Maybe `n = pos[1] + 1`, `if (stack_size < n)`, `stack.push(stack[n -1])` ?

Reply:
Done